client/me/purchases/manage-purchase/purchase-meta-expiration.tsx

```diff
@@ -56,15 +57,20 @@ function PurchaseMetaExpiration( {
 	const isAutorenewalEnabled = purchase?.isAutoRenewEnabled ?? false;
 	const isJetpackPurchaseUsingPrimaryCancellationFlow =
 		isJetpackPurchase && config.isEnabled( 'jetpack/cancel-through-main-flow' );
-	const hideAutoRenew =
-		purchase &&
-		JETPACK_LEGACY_PLANS.some( ( plan ) => plan === purchase.productSlug ) &&
-		! isRenewable( purchase );
 
-	if ( ! purchase || isDomainTransfer( purchase ) || purchase?.isInAppPurchase ) {
+	if (
+		! purchase ||
+		isDomainTransfer( purchase ) ||
+		purchase?.isInAppPurchase ||
+		PRODUCT_AKISMET_FREE === purchase?.productSlug
```

Comment:
Can we add the `isAkismetFreeProduct` function you created above to a helper or something, so we could reuse it here and the other places you did a similar check? There's a package called `calypso-products` that we've used for functions like that.